src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/controllers/PingController.java

```diff
@@ -1,21 +1,21 @@
-package br.com.mercadolivre.bootcampw2.grupo11.socialmeli.controllers;
-
-import br.com.mercadolivre.bootcampw2.grupo11.socialmeli.services.TestService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
-
-@RestController
-public class PingController {
-
-
-    @Autowired
-    private TestService service;
-
-    @GetMapping("/ping")
-    public ResponseEntity<String> ping() {
-        service.test();
-        return ResponseEntity.ok("pong");
-    }
-}
+//package br.com.mercadolivre.bootcampw2.grupo11.socialmeli.controllers;
```

Comment:
I think that you should keep ping controller and remove the service call.
Or just delete the file